### PR TITLE
Annexed Philippines, removal of minor bomber IC reductions

### PR DESCRIPTION
**Pull request by RH-131**

### CODE REVIEW

**Comment by Narizna** [comment]
so this is removing the IC reductions for bombers from various ideas instead of giving minors an IC cost debuff Idea? just want to clarify as I think the change in this PR is better than adding an idea.

**Review by Mookubooku — CHANGES_REQUESTED** [review]
Naval changes and USA annexing Philippines are fine. Needing minor plane production is not.

File: common/ideas/romania.txt [hunk 1]
```diff
@@ -1188,7 +1172,7 @@ ideas = {
 
 			equipment_bonus = {
 				fighter_equipment = {
-					build_cost_ic = -0.08
+					build_cost_ic = -0.12
```
Comment (GrenadierSchubes) [inline common/ideas/romania.txt:1175]: why does romania now get 12% production cost bonus when all the minors were gutted?

**Comment by Ruling-0** [comment]
I have to agree with Mooku, and recommend making the plane IC changes it's own PR so it can be further discussed.

**Pull request closed without merging**